### PR TITLE
[TBL-277] Default organization PR template

### PR DESCRIPTION
**Pull request by stefanklokgieters**
See https://studyportals.atlassian.net/browse/TBL-277

### CODE REVIEW

**Review by jhuys — APPROVED** [review]
I find it surprising that this pull request, which introduces i.a. a pull request template, does not have a description that follows the pull request template 😅

**Comment by stefanklokgieters** [comment]
> I find it surprising that this pull request, which introduces i.a. a pull request template, does not have a description that follows the pull request template 😅

No longer after merging this! But indeed, I was expecting at least one person to point that out haha 👯

**Review by dandobrescu — APPROVED** [review]

**Review by ragalob — APPROVED** [review]

**Review by Stormyy — APPROVED** [review]

**Review by MartySP — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
--
--
+## What Jira issue is your PR related to?
+_e.g. [TBL-123](https://studyportals.atlassian.net/browse/TBL-123)_
```
Comment (borismo) [inline PULL_REQUEST_TEMPLATE.md:2]: Sorry, late to the party, but is it a good idea to have this URL in a public repo?

Comment (stefanklokgieters) [reply PULL_REQUEST_TEMPLATE.md:2]: There is no access to it externally, so should be fine. But good that you paid attention!

Comment (dandobrescu) [reply PULL_REQUEST_TEMPLATE.md:2]: I don't think it's too much of a problem to leak the issue ID, as even if somebody gains access to Atlassian's database they still would be able to do much with the information they have from JIRA.

**Pull request merged**